packages/@aws-cdk/aws-appmesh/lib/tls-client-policy.ts

```diff
@@ -1,9 +1,18 @@
+import { TlsCertificate } from './tls-certificate';
 import { TlsValidation } from './tls-validation';
 
 /**
  * Represents the properties needed to define client policy
  */
 export interface TlsClientPolicy {
+  /**
```

Comment:
Let's move this new property to the end of this interface, instead of it being at the beginning.

Reply:
Addressed.